xmodule/video_module/transcripts_utils.py

```diff
@@ -166,15 +228,15 @@ def get_transcripts_from_youtube(youtube_id, settings, i18n, youtube_transcript_
 
     utf8_parser = etree.XMLParser(encoding='utf-8')
 
-    youtube_text_api = copy.deepcopy(settings.YOUTUBE['TEXT_API'])
-    youtube_text_api['params']['v'] = youtube_id
-    # if the transcript name is not empty on youtube server we have to pass
-    # name param in url in order to get transcript
-    # example http://video.google.com/timedtext?lang=en&v={VideoId}&name={transcript_name}
-    youtube_transcript_name = youtube_video_transcript_name(youtube_text_api)
-    if youtube_transcript_name:
-        youtube_text_api['params']['name'] = youtube_transcript_name
-    data = requests.get('http://' + youtube_text_api['url'], params=youtube_text_api['params'])
```

Comment:
What I don't understand here is why this wasn't working. Instead of making this work, you implemented a new way of doing it? Can you explain a bit more, please?